### DISCUSSION TITLE
开发工具上的swap为什么无法使用了

### DISCUSSION THREAD

**Discussion by wufeng1125** [mangodager/smartx-dotnet#16] [Q&A]
开发者工具上的swap不能用了，查询不到交易兑！看不到流动池

**Comment by mangodager** [comment]
最新SmartxApp版本可以使用了

**Reply by zoukz115** [reply]
顾左右而言他。人家说开发者工具，你们非要回到钱包。

**Reply by mangodager** [reply]
> 顾左右而言他。人家说开发者工具，你们非要回到钱包。

开发者工具也是可以使用的

**Reply by mangodager** [reply]
不过开发者工具因为是单节点，所以单节点不出块的时候会有丢失现象，这个时候重发即可。
